### ISSUE TITLE
CodeBlock - aria attributes need updating in examples

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#11386]
**Describe the enhancement or change**
For both examples, the aria-label of the buttons should be updated:

- the copy button should have an aria-label that conveys what will be copied, e.g. `Copy to clipboard basic example code block`. 
- ~The "play" button aria-label shouldn't have the word "icon" in it and either just be "Play" or follow the ClipboardCopy additional actions example and use "Run in web terminal" -- this should also have a tooltip aplied to it.~

For the expandable example, the `aria-labelledby` on the expandable content points to a non-existent ID. Passing the same `toggleId` to both ExpandableSection and ExpandableSectionToggle will resolve this.

**Is this request originating from a Red Hat product team? If so, which ones and is there any sort of deadline for this enhancement?**

**Any other information?**

**Comment by Mash707** [comment]
@thatblindgeye I would like to work on this.

**Comment by Mash707** [comment]
@thatblindgeye some changes were made regarding this in #11583, could you check once. Thank You.

**Comment by thatblindgeye** [comment]
@Mash707 yep that second bullet is no longer relevant here, thank you!
